packages/declaration/lua/language-server/Object.def.lua

```diff
@@ -681,10 +681,11 @@ function Object.getVelocity() end
 ---@return tts__ScriptingTrigger[]
 function Object.getZones() end
 
+---@param color tts__ColorShape|string
```

Comment:
What does that parameter do in this case? Seems kinda odd to have one there, though it's also in the API docs. ^^" Is it optional or required? If optional, could be written as `color?`.

Reply:
It just removes the specific highlight color instead of all. The API doesn't say so, but I assume it is optional.